src/Api/Discretionary.ts

```diff
@@ -0,0 +1,83 @@
+import {
+    DISCRETIONARY_TRANSACTION_TYPE,
+    IDiscretionaryCampaign,
+    IDiscretionaryCampaignFilter,
+    IDiscretionaryIssuePayload,
+    IDiscretionaryLedgerTransaction,
+    IDiscretionaryResponse,
+    IDiscretionaryTransaction,
+    IDiscretionaryTransactionFilter,
+    IPodiumPromise,
+    ISettings,
+    SORT_DIRECTION,
+    SORT_FIELD,
+} from '../../types'
+import {Resource} from '../Podium/Resource'
+import {Paginator} from '../Podium/Paginator'
+import {Filter} from '../Podium/Filter'
+
+export class Discretionary extends Resource {
+
+    constructor(settings: ISettings) {
+        super(settings)
+    }
+
+    // Retrieves single campaign.
+    public GetCampaign(campaignId: number): IPodiumPromise<IDiscretionaryCampaign> {
+        super.SetResource(`campaign/discretionary/${campaignId}`)
```

Comment:
This is not necessary; inherited from Resource

Reply:
Resource overwrite makes it necessary (i think)